lib/comgr/src/comgr-compiler.cpp

```diff
@@ -581,6 +582,23 @@ InProcessDriver::InProcessDriver(raw_ostream &DiagOS)
 }
 
 amd_comgr_status_t InProcessDriver::Execute(ArrayRef<const char *> Args) {
+  // For HIP compilation, we launch a process.
+  bool IsHIP = false;
```

Comment:
I would prefer this be derived more directly from `ActionInfo->language`; `Args` can contain arbitrary user options at this point, and implicitly updating the source language based on them might be unexpected.